test/Vcs/Git/GitVcsTest.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.IO;
+using Skarp.Version.Cli.Vcs.Git;
+using Xunit;
+
+namespace Skarp.Version.Cli.Test.Vcs.Git
+{
+    public class GitVcsTest : IClassFixture<GitVcsFixture>
+    {
+        private readonly GitVcsFixture _fixture;
+
+
+        public GitVcsTest(GitVcsFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void ReturnsProperToolname()
+        {
+            Assert.Equal("git", _fixture.Vcs.ToolName());
+        }
+
+        [Fact]
+        public void DetectingGitOnMachineWorks()
```

Comment:
**question**: can we make a negative test for this as well?

I am not sure how to go about it, but currently the implementation could just be `return true` and the test would pass :)

Reply:
I guess we could but I'm not sure how easily that is done - I do agree with your comment though.

running it on a machine without git in CI will not work, as it won't be able to get the source code - but perhaps altering the path for the `process info` we use to launch the tool will work.